validator/testdata/feature_tests/forms.out

```diff
@@ -130,6 +130,17 @@ feature_tests/forms.html:105:2 Invalid URL protocol 'http:' for attribute 'actio
 feature_tests/forms.html:125:2 The attribute 'target' in tag 'form' is set to the invalid value '_new'. (see https://amp.dev/documentation/components/amp-form)
 |      <input type="submit" value="Subscribe">
 |    </form>
+|    <!-- Valid: input with form attribute can be outside the form. -->
+|    <input form="my-form3" type="text" name="name">
+>>   ^~~~~~~~~
+feature_tests/forms.html:129:2 The attribute 'form' may not appear in tag 'input'. (see https://amp.dev/documentation/components/amp-form/)
```

Comment:
Does this need to be updated with `gulp validator --update-tests` still?

Reply:
Resolving as I believe there shouldn't be a new tagspec for this but included in input-common-attr attribute list.